docs/introduction/actions.md

```diff
@@ -16,21 +13,12 @@ return ReceivedNewPhoneNumber
 
 We can then `dispatch` an `action` to our `store` via [`Store:dispatch`](../api-reference.md#storedispatch) like so:
 
-!!! info
-	This example assumes that you've successfully [installed Rodux](installation.md) into `ReplicatedStorage` and placed the sample code above in `ReplicatedStorage.Actions.ReceivedNewPhoneNumber`!
-
 ```lua
-local ReplicatedStorage = game:GetService("ReplicatedStorage")
-
-local Rodux = require(ReplicatedStorage.Rodux)
-
-local ReceivedNewPhoneNumber = require(ReplicatedStorage.Actions.ReceivedNewPhoneNumber)
-
 local store = Store.new(function(currentState, action)
 	-- The body of your reducer
 end)
 
-store:dispatch(ReceivedNewPhoneNumber("19876543"))
+store:dispatch(ReceivedNewPhoneNumber("12345678"))
```

Comment:
This is the *ultimate* nitpick, but what phone numbers are 8 digits??

Reply:
1 digit for the country code, 7 digits for the rest.

Reply:
Lol, good point X)!! I'll fix it.